### DISCUSSION TITLE
CMIP7 Data Request v1.0: now available!

### DISCUSSION THREAD

**Discussion by adagj** [NorESMhub/noresm3_dev_simulations#53] [Announcements]
The CMIP7 Data Request structure differs from previous phases. Modelling centres and MIPs are invited to provide feedback on v1.0beta. More information is available here: https://wcrp-cmip.org/cmip7-data-request-v1-0beta/

The full Data Request can be explored online at https://bit.ly/CMIP7-DReq-v1_0beta

**Comment by adagj** [comment]
From CMIP-IPO:
Dear MIP and Modelling Centre contacts,

The Data Request Task Team are holding a virtual launch event following the [Data Request v1.0 release](https://wcrp-cmip.org/cmip7/cmip7-data-request/). The event is aimed at MIPs and Modelling Centres, though anyone from the CMIP Community is invited to register.
 
During the session, the Task Team will present an overview of the request, including descriptions of the changes in structure since CMIP6, an overview of the information and tools available to you, and an introduction to the CMIP7 Data Request software package. There will be opportunities for questions and discussion throughout the session.

The Data Request Task Team is also interested in how MIPs and Modelling Centres would like us to engage with you in the coming months, especially with two future Data Request versions planned for release in January (v1.1) and March (v1.2). Please come ready to participate in this interactive discussion.

Please note, this meeting is being held across two different sessions. The two meetings are being held to enable participation from colleagues working in different time zones. You only need to attend one of the two meetings.

Register for Session 1: [Tuesday 26 November, 17:00-18:00 UTC](https://wcrp-cmip.org/event/data-request-v1-0-launch-1/)

Register for Session 2: [Wednesday 27 November, 08:00-09:00 UTC](https://wcrp-cmip.org/event/data-request-v1-0-launch-2/)

**Comment by MichaelSchulzMETNO** [comment]
Dear MIPs and Modelling Centres,
 
We are pleased to [release v1.0 of the CMIP7 Data Request](https://wcrp-cmip.org/cmip7-data-request-v1-0). There are two key components of the request that you can access and review from today: 
[the online database (hosted on Airtable),](https://bit.ly/CMIP7-DReq-v1_0)
[the software package hosted on GitHub](https://github.com/CMIP-Data-Request/CMIP7_DReq_Software). 
 
All information about accessing and using the request is available on our website at https://wcrp-cmip.org/cmip7-data-request-v1-0 for future reference, including links to guidance documents – please read this carefully.
 
Please feel free to continue to provide feedback using the Excel spreadsheets and/or Variable Comment forms circulated [during the v1.0beta release](https://wcrp-cmip.org/cmip7-data-request-v1-0beta/). These will continuously be incorporated into the request.
 
We would like to thank the thematic author teams for all their hard work developing the request.
 
Best wishes,
CMIP IPO
